engine/src/main/java/org/terasology/rendering/dag/nodes/DownSampleSceneAndUpdateExposureNode.java

```diff
@@ -206,10 +239,10 @@ private void downSampleSceneInto1x1pixelsBuffer() {
      * only if eye adaptation is enabled: an NPE would be thrown only in that case.
      */
     private void obtainStaticFBOs() {
-        downSampledScene[4] = frameBuffersManager.getFBO("scene16");
-        downSampledScene[3] = frameBuffersManager.getFBO("scene8");
-        downSampledScene[2] = frameBuffersManager.getFBO("scene4");
-        downSampledScene[1] = frameBuffersManager.getFBO("scene2");
-        downSampledScene[0] = frameBuffersManager.getFBO("scene1");
+        downSampledScene[4] = staticFBM.getFBO(SCENE_16_URN);
+        downSampledScene[3] = staticFBM.getFBO(SCENE_8_URN);
+        downSampledScene[2] = staticFBM.getFBO(SCENE_4_URN);
+        downSampledScene[1] = staticFBM.getFBO(SCENE_2_URN);
+        downSampledScene[0] = staticFBM.getFBO(SCENE_1_URN);
```

Comment:
No, it's not used. Do you think we should remove staticFBM? Apart from sharing with other nodes, FBM makes sure that the FBO is disposed after the node is removed from the graph. If we decide not to publish it, it'll be node's responsibility to do so.